test/pytest/test_peer.py

```diff
@@ -70,8 +70,9 @@ async def test_init_failure(rule, init_type, monkeypatch, transport):
     await transport.check_open('test', problem='protocol-error')
 
 
+# this doesn't use await, but requires an event loop
 @pytest.mark.asyncio
-async def test_immediate_shutdown(rule):
+async def test_immediate_shutdown(rule):  # noqa: RUF029
```

Comment:
I talked this over with @allisonkarlitskaya in chat, and she agreed to this.

Reply:
Sounds good to me!